scripts/nrepl-test.py

```diff
@@ -0,0 +1,16 @@
+import socket
+import time
+clientSocket = socket.socket(socket.AF_INET, socket.SOCK_STREAM);
+clientSocket.connect(("127.0.0.1", 8181))
+print(clientSocket)
+data = clientSocket.recv(1024)
+print(data.decode())
+
+form = "(repl-help)"
+
```

Comment:
python actually has a nice built-in library for creating stuff like this: https://docs.python.org/3/library/struct.html
```
import struct
header = struct.pack('<II', len(form), 10)
```
could replace the `b'\x0B\x00\x00\x00\x0A\x00\x00\x00'`.